test/io/panzone/real-writeln.good

```diff
@@ -0,0 +1,8 @@
+a = 1.0
+b = 10.0
+c = 100.0
+d = 1000.0
+e = 10000.0
+f = 1.00000e+05
+g = 1e+06
```

Comment:
Why does f print out with 1.00000e+05 and not 1+05 ? Can you get it to print the simpler version?